src/MTracks.hpp

```diff
@@ -96,7 +96,7 @@ class MTracks : public QObject
     inline int nbTracks() const {
         if(!_tracks || _status != MTracks::Ready)
             return 0;
-        return _tracks->size();
+        return (int)_tracks->size();
```

Comment:
same here, I'd rather return a std::size_t that represents a positive number.

Reply:
Done as well.